ci/coverage-report-generator/run.sh

```diff
@@ -15,12 +15,12 @@ REPORT_FILE="${1:-${REPO_ROOT}/coverage/.resultset.json}"
 
 docker run \
     --rm \
-    -v "${REPO_ROOT}":"${REPO_ROOT}" \
-    -w "${REPO_ROOT}/ci/coverage-report-generator" \
+    --volume "${REPO_ROOT}":"${REPO_ROOT}" \
+    --workdir "${REPO_ROOT}/ci/coverage-report-generator" \
     "${IMAGE}" \
-        bash -cex "bundle install --path gems
-                   bundle list
-                   bundle exec ./generate_report.rb \
-                       ${REPO_ROOT} \
-                       ${REPORT_FILE}
-                  "
\ No newline at end of file
+    bash -cex "
+      gem install bundler -v 2.1.4 
+      bundle config set path 'gems'
```

Comment:
@hughsaunders Wrote this, I don't know what the original reason was.  The `bundle config set path 'gems'` is doing the same work as `--path gems` in the previous version of the diff.  It stores the downloaded gems in the `gems` subdir of this dir.  Maybe to avoid conflicts with the parent ruby install.  I'm not entirely sure.

Reply:
It was to avoid a deprecation notice about `--path`

```[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set path```

Reply:
I guess the big question for me is not about why it's different vs `--path` but why do we need to set the gem path in the first place..

Reply:
@hughsaunders Do you know the reason?  I do not...

Reply:
This isn't a blocker though so I'll mark it resolved but it would be good to hear a response on it @hughsaunders